source/Guides/whitelist.htm.erb

```diff
@@ -58,10 +58,7 @@ description: "The Gridcoin whitelist is a list of community-approved BOINC proje
                                 Sponsor
                             </th>
                             <th>
-                                CPU
-                            </th>
-                            <th>
-                                GPU
+                                Compatibility
```

Comment:
Why this change?

Reply:
Ahh that was residual changes from my whitelist update branch, missed that bit